### PR TITLE
feat: Added support for uploading pictures to Alibaba Cloud

### PR DESCRIPTION
**Pull request by ximing**
| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     | yes
| BC breaks?       | no
| Deprecations?    | no
| New tests added? | not needed
| License          | MIT

### Description
[feature]

Added support for uploading pictures to Alibaba Cloud
Regarding OSS, you can check the links below, most of which are close to S3 object storage.
https://www.alibabacloud.com/getting-started/learningpath/oss?spm=a3c0i.7950270.1167927.5.3a71ab91C6Ados

--

### CODE REVIEW

File: src/renderer/prefComponents/imageUploader/services.js [hunk 1]
```diff
@@ -35,6 +35,14 @@ const services = {
 
     // Currently a non-persistent value
     agreedToLegalNotices: false
+  },
+
+  alioss: {
+    name: 'alioss',
+    isGdprCompliant: true,
+    privacyUrl: '',
+    tosUrl: '',
```
Comment (fxha) [inline src/renderer/prefComponents/imageUploader/services.js]: Please add the privacy police and TOS URLs here.

File: src/renderer/store/preferences.js [hunk 2]
```diff
@@ -87,6 +87,12 @@ const state = {
       owner: '',
       repo: '',
       branch: ''
+    },
+    alioss: {
+      accessKey: '',
+      accessSecret: '',
```
Comment (fxha) [inline src/renderer/store/preferences.js]: The secret key should be saved by `keytar` like the Github token.

**Comment by ximing** [comment]
OK，I will try to use S3 protocol to implement a more general image upload adapter

**Comment by Jocs** [comment]
Thank you very much for your PR, but he may not be our main direction at the moment, and we hope to support it through plugins, so this issue is closed first, and I hope you can submit other PRs to improve Mark Text, thank you again

**Comment by pengxianggui** [comment]
> Thank you very much for your PR, but he may not be our main direction at the moment, and we hope to support it through plugins, so this issue is closed first, and I hope you can submit other PRs to improve Mark Text, thank you again

Any plugin development manual provide? I want upload image to ali-oss, support local choice、Screenshots paste.

**Pull request closed without merging**